package.json

```diff
@@ -4,17 +4,18 @@
   "description": "Multiplayer code editor system",
   "main": "src/index.ts",
   "type": "module",
-  "bin": "src/server/index.js",
+  "bin": "src/server/cli.js",
```

Comment:
CLI changes in the deep linking PR is unfortunate.

Reply:
sorry for mixing up my branches, I rebased on main and soon will change also #422 to mitigate the change.

Reply:
Thank you so much!